src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+const WAIT = 3000;
+const LOGO_EVENT = 'click';
+
+const logo = document.querySelector('.logo');
+const elemDone = document.createElement('div');
+
+elemDone.classList.add('message');
+
+const elemDeny = elemDone.cloneNode(true);
+
+elemDeny.classList.add('error-message');
+
+elemDone.textContent = 'Promise was resolved!';
+elemDeny.textContent = 'Promise was rejected!';
+
+const promise1 = new Promise((resolve) => {
+  logo.addEventListener(LOGO_EVENT, () => {
```

Comment:
The event listener for the logo element is set up correctly, but ensure that the '.logo' class is present in the HTML. If the element is not found, `logo` will be `null`, and adding an event listener will cause an error. Consider adding a check to ensure `logo` is not `null` before adding the event listener.